build-scripts/runtime_lib.cmake

```diff
@@ -84,6 +84,11 @@ if (WAMR_BUILD_LIB_PTHREAD EQUAL 1)
     set (WAMR_BUILD_SHARED_MEMORY 1)
 endif ()
 
+if (WAMR_BUILD_DEBUG_ENGINE EQUAL 1)
```

Comment:
Using DEBUG_ENGINE for interpreter and DEBGU_INFO for AOT seems not friendly to developer, how about chaning to DEBUG_INTERP and DEBUG_AOT instead?